### DISCUSSION TITLE
Plans for Amy?

### DISCUSSION THREAD

**Discussion by rocketsailor** [Techokami/SonicWorldsNext#35] [General]
Hello! I noticed adding Amy is a low priority on the to-do list. Just wanting to clarify before I start modifying the project myself: is the plan to wait for Origins Plus to release?

**Comment by Techokami** [comment]
Yeah, we're waiting on Origins Plus to see how Amy works there, and use that implementation as the basis for our version. The implementation used in Sonic Worlds Delta, which is based on Advance 1, will most likely be relegated to a new character with a much lower priority. However at this time our priorities are with getting the code working again with Godot 4.

**Reply by rocketsailor** [reply]
okay. thanks for the reply!
